config.toml

```diff
@@ -24,7 +24,6 @@ home = [ "HTML", "REDIRECTS" ]
   mediumHandle   = "jaegertracing"
   gitterHandle   = "jaegertracing"
   openGraphImage = "img/jaeger-icon-color.png"
-  newsletterTitle = "October 2018 project update"
```

Comment:
Is this change intentional?

Reply:
Yes, cleaning up a leftover from earlier change that gets this title directly from medium